lib/apiservers/portlayer/swagger.json

```diff
@@ -2721,8 +2763,7 @@
 					"$ref": "#/definitions/ReservationConfig"
 				},
 				"storageSize": {
-					"type": "integer",
-					"format": "int64"
+					"type": "integer"
```

Comment:
After our slack conversation, I see that integer now defaults to int64 in the latest Swagger generator.